### DISCUSSION TITLE
(mini.pick) how pick dot files (hidden files)?

### DISCUSSION THREAD

**Discussion by mainendra** [nvim-mini/mini.nvim#643] [Q&A] [answered]
Is there any way we can list dot files using `Pick files`?
e.g. `if tool == 'rg' then return { 'rg', '--files', '--no-follow', '--color=never', '--hidden' } end`

**Comment by echasnovski** [comment, marked as answer]
There are at least two choices here:
- [Customize `rg` CLI tool](https://github.com/echasnovski/mini.nvim/blob/a118a964c94543c06d8b1f2f7542535dd2e19d36/doc/mini-pick.txt#L79-L84) to search hidden files by default. This is what I use.
- Create custom picker using [`builtin.cli()`](https://github.com/echasnovski/mini.nvim/blob/a118a964c94543c06d8b1f2f7542535dd2e19d36/doc/mini-pick.txt#L1141) to call the command you want.
